AEPCore/Sources/core/CoreConstants.swift

```diff
@@ -39,6 +39,7 @@ enum CoreConstants {
         static let REFRESH_RULES = "Refresh Rules"
         static let SET_PUSH_IDENTIFIER = "SetPushIdentifier"
         static let SET_ADVERTISING_IDENTIFIER = "SetAdvertisingIdentifier"
+        static let REQUEST_RESET = "Identities Request Reset"
```

Comment:
nit: "REQUEST_IDENTITIES_RESET", just in case we add another event to request a reset of something else.